float.js

```diff
@@ -199,9 +201,14 @@ const addButtons = function() {
         buttonText.innerText = 'Get Float';
         getFloatButton.appendChild(buttonText);
 
-        let messageDiv = document.createElement('div');
-        messageDiv.classList.add('floatmessage');
-        buttonDiv.appendChild(messageDiv);
+        // Create divs the following class names and append them to the button div
+        let divClassNames = ['floatmessage', 'itemfloat', 'itemseed'];
+
+        for (let className of divClassNames) {
+            let div = document.createElement('div');
```

Comment:
Sadly the `p` tag has styling for padding to the top and bottom of the element. We'd have to customize styling for it either by using JS or injecting a stylesheet. Semantically it's better, but I don't think it is worth the additional manual styling needed.

Just to make sure I understand which tags you want to change, I'm referring to this:
```
<div id="item_284126190785063316_floatdiv" style="display: inline; text-align: left;">
    <p class="itemfloat">Float: 0.2196776419878006</p>
    <p class="itemseed">Paint Seed: 789</p>
</div>
```